### DISCUSSION TITLE
How to disable Annotation Editing?

### DISCUSSION THREAD

**Discussion by sglbl** [sumatrapdfreader/sumatrapdf#2331] [Q&A] [answered]
Hello,
To annotate(or add images) pdf documents, I use Drawboard Pdf. When I open that pdf file with SumatraPDF, if I mistakenly click/shift that image, it removes/make it disappear image from PDF. So, is there an option to turn of annotation editing?

**Comment by GitHubRulesOK** [comment]
Currently no (anoying I know and the change to custom annotations has already been noted as an issue)

You would either need to use an older version 3.2 (without annotation)
OR SIMPLER close file without save and reopen it

**Comment by kjk** [comment]
Currently it's not possible and I'm not convinced that accidental annotation editing is common enough to warrant adding a way to disable annotation editing.

**Reply by GitHubRulesOK** [reply]
@kjk 
I found that if an image say a stamped signature is included as a custom annotation it was destroyed by the accidental replacement appearance, so currently SumatraPDF is not including widget type or more than basic stock stamps like "Paid"
thus not distinguishing custom stamps (unsure if custom and stock stamps are defined, as seperate types)

**Reply by kjk** [reply]
Please open an issue with repro file and steps-by-step repro instructions. Bugs reported in discussion will almost certainly be forgotten.

**Reply by GitHubRulesOK** [reply]
@kjk One example I was thinking of is in  https://github.com/sumatrapdfreader/sumatrapdf/issues/1681 as mentioned in https://github.com/sumatrapdfreader/sumatrapdf/issues/2000 sample there is https://github.com/sumatrapdfreader/sumatrapdf/files/4909953/pmi_sample3.pdf

Personally I have no issue with moving an ink string or any other shaped object to different XY (or better yet adjust dx dy) , since that may be desirable, it is just the destruction of content when slightly moved that is my issue.

**Reply by bantab** [reply]
It looks like you may be correct that it's not common enough for most people. However, I will say that in my use case, which involves viewing a lot of pdf CAD drawings, I am constantly dragging annotation elements. I can close without saving, but having the dialogue box pop up pretty much every time I work with a drawing does become annoying. I'd go back to a previous version, but the new multiple tabbed window feature is way too useful. Having it as an "Advanced option" would be awesome, but I understand it's probably low priority.

**Reply by PhotoBoardSystem** [reply]
Hello,
I too am very bothered by this problem. A program I developed organizes hundreds of photos into a single PDF and Sumatra is appreciated for being very quick to hover and zoom those photos. Can you add an option to find the same comfort as V3.2 please?

**Comment by brianbienvenu** [comment]
I constantly find SumatraPDF trying to mistakenly edit engineering drawings (containing a lot of overlapping lines/anotations/text). It often crashes in the process.

Using click and drag to move the view around is essential, but basically broken by the editing functionality as it stands today.

This is absolutely a feature which should be implemented.

**Comment by kjk** [comment]
I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/3261 to track this.

Most likely will require pressing additional key e.g. Shift or Ctrl to move an annotation with a mouse to prevent accidental moving of annotations.

**Reply by PhotoBoardSystem** [reply]
Hi !
I approve! It would be wonderful if the annotations only move if Shift or Alt key is pressed (and not Ctrl which is used for zooming).
THANKS.

**Comment by charygao** [comment]
same require +1 ,  add the  disable Annotation functions command settings please.THANKS.

**Comment by kjk** [comment]
It's now harder to accidently move annotations: you have to hold Ctrl while mouse dragging.

Please try and let me know if more needs to be done (like completely disabling any annotations functionality).

**Reply by PhotoBoardSystem** [reply]
Hi !
The problem is now resolved with the pre-release v3.5.15699 64bits. Thank you so much !

**Comment by sglbl** [comment, marked as answer]
Today I installed `v3.5.2` and the issue seems to be fixed. I have been using `v3.2` for years just because of this irritating thing. Thank you all the contributors.

**Reply by GitHubRulesOK** [reply]
You could also try 3.6.pre-release for more  extra features like disable commands though unsure if that can matches this issue apart from above comment that CTRL key is usually needed to shift any comments.
